### PR TITLE
Added BloodClotSubType enum

### PR DESCRIPTION
**Pull request by KyojoH**

### CODE REVIEW

File: typings/unofficial/enumsSubTypes.d.ts [hunk 1]
```diff
@@ -1,3 +1,16 @@
+/** For EntityType.ENTITY_FAMILIAR (3), FamiliarVariant.BLOOD_BABY(238) */
```
Comment (Zamiell) [inline typings/unofficial/enumsSubTypes.d.ts:1]: bug, you forgot a space

File: typings/unofficial/enumsSubTypes.d.ts [hunk 2]
```diff
@@ -1,3 +1,16 @@
...
+declare const enum BloodClotSubType {
+  RED = 0,
+  SOUL = 1,
+  BLACK = 2,
+  ETERNAL = 3,
+  GOLD = 4,
+  BONE = 5,
+  ROTTEN = 6,
+  /** Spawned by the Blood Clot trinket; cannot be turned into health by Sumportium */
```
Comment (Zamiell) [inline typings/unofficial/enumsSubTypes.d.ts:10]: jsdoc must be complete sentences

**Comment by Zamiell** [comment]
please dont submit PRs until they pass the linter first

File: typings/unofficial/enumsSubTypes.d.ts (hunk 1 above)
Comment (KyojoH) [reply typings/unofficial/enumsSubTypes.d.ts]: fixed

File: typings/unofficial/enumsSubTypes.d.ts (hunk 2 above)
Comment (KyojoH) [reply typings/unofficial/enumsSubTypes.d.ts]: fixed i think

**Comment by Zamiell** [comment]
why did you add it to this spot below dip 201? shouldn't it be in order of increasing subtype?

**Comment by KyojoH** [comment]
i was adding it in alphabetical order

**Comment by Zamiell** [comment]
much better now, thanks

**Pull request merged**